Formula/secman.rb

```diff
@@ -0,0 +1,26 @@
+class Secman < Formula
+  desc "👊 Human-friendly and amazing secrets manager"
+  homepage "https://secman.dev/docs/cli"
+  url "https://cli-files.secman.dev/secman-v6.0.71/secman-v6.0.71.tar.gz"
+  sha256 "2a9c03df9b444e9c110b640c2e22ef4db147623cb728f7b2c68f2bc488d0e230"
+  license "MIT"
+
+  livecheck do
+    url :homepage
+    regex(/href=.*?aamath[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "scmn-dev/secman/sm-node"
+
+  def install
+    inreplace "bin/secman", /^CLIENT_HOME=/, "export SECMAN_OCLIF_CLIENT_HOME=#{lib/"client"}\nCLIENT_HOME="
+    inreplace "bin/secman", "\"$DIR/node\"", Formula["sm-node"].opt_bin/"node"
+    libexec.install Dir["*"]
+    bin.install_symlink libexec/"bin/secman"
+    ENV["SM_PROVIDER"] = "brew"
```

Comment:
This line doesn't do anything.

Reply:
do you mean `ENV["SM_PROVIDER"] = "brew"` ? @carlocab

Reply:
Yes. You're setting an env variable for the build environment, but nothing else occurs after that to use the variable you've set.